app/src/main/res/values/strings.xml

```diff
@@ -911,6 +911,9 @@
     <string name="location_enabled">Location: Enabled </string>
     <string name="location_disabled">Location: Disabled </string>
 
+    <string name="prmsn_granted_storage">Permission Granted for storage</string>
```

Comment:
Shouldn't it be simple letters for `Granted`?

Reply:
I didnt understand?

Reply:
It's grammar.